torch/_torch_docs.py

```diff
@@ -8625,6 +8625,28 @@ def merge_dicts(*dicts):
 
 If :attr:`upper` is ``False``, then lower triangular portion is used.
 
+.. warning::
+
+    :func:`torch.symeig` is deprecated in favor of :func:`torch.linalg.eigh`
+    and will be removed in a future PyTorch release. The default behavior has changed
+    from using the upper triangular portion of the matrix by default to using the
+    lower triangular portion. This should only matter if this function is used with
+    non-symmetric matrices, which is not recommended.
```

Comment:
"not recommended" parenethetical here should probably be killed

Reply:
I'll delete the whole sentence since, as described in https://github.com/pytorch/pytorch/pull/57732#discussion_r629202509 we do not really support this behaviour.